### PR TITLE
Create Skill class (idk how this works)

### PR DESCRIPTION
**Pull request by seanjhardy**
Create ~skill test command
Refactor the code to conform to ESLint
Add assets file and test assets
Setup environment for deployment

**Please describe the changes this PR makes and why it should be merged, as well as any and all proof of successful tests:**

I added a thing and some helpful functions, pls be nice o_o

**Semantic versioning classification:**

- [x] This PR changes the framework's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to README, etc.

### CODE REVIEW

**Review by swatarianess — APPROVED** [review]
Good job! Looks neet.

File: commands/conf.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-FOR GUILD SETTINGS SEE set.js !
+FOR GUILD bSETTINGS SEE set.js !
```
Comment (swatarianess) [inline commands/conf.js:2]: Fixing mispelling.

File: config.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ const config = {
   * DO NOT REMOVE THIS BEFORE YOUR BOT IS LOADED AND FUNCTIONAL.
   */
   "defaultSettings" : {
-    "prefix": "~",
+    "prefix": "s/",
```
Comment (swatarianess) [inline config.js:27]: why `s/` instead of `~` ?

**Review by swatarianess — CHANGES_REQUESTED** [review]
Added 2 comments :)

File: config.js (hunk 2 above)
Comment (iJynx) [reply config.js:27]: We have ~ for jeffrey bot and every second person doesn't know the difference betwen ~ and -

**Review by iJynx — APPROVED** [review]
Looks alright, i'll do the rest later.

**Review by swatarianess — APPROVED** [review]

**Pull request merged**